### ISSUE TITLE
Image caption field limited length

### ISSUE THREAD

**Issue by kookster** [PRX/cms.prx.org#572]
Got an error in NR 

`Api::StoryImagesController#createMysql2::Error: Data too long for column 'caption' at row 1`

Looks like someone put a very long caption on an image, and it failed to save.
We should either increase the field size, or maybe we need to validate on the FE or in the model before save.

**Comment by cavis** [comment]
Yep, on the Publish side, it looks like we have no validators on [the image model](https://github.com/PRX/publish.prx.org/blob/master/src/app/shared/model/image.model.ts).  Whereas the [audio model](https://github.com/PRX/styleguide.prx.org/blob/master/projects/ngx-prx-styleguide/src/lib/upload/model/audio-file.model.ts#L25) over in Styleguide _does_ have validators.
